operations/src/main/java/com/nextdoor/bender/operation/delete/DeleteFieldOperation.java

```diff
@@ -0,0 +1,35 @@
+package com.nextdoor.bender.operation.delete;
+
+import com.nextdoor.bender.InternalEvent;
+import com.nextdoor.bender.deserializer.DeserializedEvent;
+import com.nextdoor.bender.operation.EventOperation;
+import com.nextdoor.bender.operation.OperationException;
+import org.apache.commons.lang3.Validate;
+
+public class DeleteFieldOperation implements EventOperation {
+    private final String key;
+
+    public DeleteFieldOperation(String key) {
+        Validate.notNull(key);
```

Comment:
This check isn't necessary. The configuration `@JsonProperty(required = true)` enforces that this field is present. If you still want to verify then it would be more appropriate to do this in the factory class before `DeleteFieldOperation` gets created.

Reply:
ah got it, forgot that would do it. I'll just delete this.